plugins/org.polarsys.capella.scenario.editor.doc/html/User Manual/Functionalities.html

```diff
@@ -270,7 +270,7 @@ <h3 id="Combined_Fragments">Combined Fragments</h3>
 				</td>
 				<td>
 					<b>alt</b> "condition A" 
-					<b>over</b> "timeline1" "timeline2"… {
+					<b>over</b> "timeline1" "timeline2"� {
```

Comment:
?

Reply:
This special char does not seem to be supported on my platform.